examples/wind/src/layers/wind-layer/wind-layer.js

```diff
@@ -0,0 +1,228 @@
+/* global Image */
+import {Layer} from 'deck.gl';
+import {GL, Model, Geometry} from 'luma.gl';
+
+import DelaunayInterpolation from '../delaunay-interpolation/delaunay-interpolation';
+import {
+  ELEVATION_DATA_IMAGE, ELEVATION_DATA_BOUNDS, ELEVATION_RANGE, LIGHT_UNIFORMS
+} from '../../defaults';
+
+import vertex from './wind-layer-vertex';
+import fragment from './wind-layer-fragment';
+
+const defaultProps = {
+  boundingBox: null,
+  originalBoundingBox: null,
+  dataBounds: null,
+  dataTextureArray: null,
+  dataTextureSize: null,
+  time: 0
+};
+
+export default class WindLayer extends Layer {
+
+  initializeState() {
+    const {gl} = this.context;
+    const {dataTextureSize, originalBoundingBox} = this.props;
+
+    // FIXME - Layer API for async loading
+    const data = {};
+    const image = new Image(584, 253);
+    image.onload = () => {
+      data.img = image;
+    };
+    image.src = ELEVATION_DATA_IMAGE;
```

Comment:
better to use the new loadTextures API, let's sync on how to integrate it with the createTextureNew() function in DelaunayInterpolation